### PR TITLE
Fix cmake minimum required version

### PR DESCRIPTION
**Pull request by miguelgaio**
Since commit 088f20... (Add export/import symbols API),
the cmake minimum required version is 2.8.12.

Update cmake minimum required version accordingly.

Signed-off-by: Miguel Gaio <miguel.gaio@intel.com>
<a href='#crh-start'></a><a href='#crh-data-%7B%22processed%22%3A%20%5B%22https%3A//github.com/01org/parameter-framework/pull/232%23issuecomment-140422829%22%2C%20%22https%3A//github.com/01org/parameter-framework/pull/232%23issuecomment-140429815%22%2C%20%22https%3A//github.com/01org/parameter-framework/pull/232%23issuecomment-140727466%22%2C%20%22https%3A//github.com/01org/parameter-framework/pull/232%23discussion_r39540215%22%2C%20%22https%3A//github.com/01org/parameter-framework/pull/232%23discussion_r39540235%22%5D%2C%20%22comments%22%3A%20%7B%22General%20Comment%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/01org/parameter-framework/pull/232%23issuecomment-140422829%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22%3A%2B1%3A%20%22%2C%20%22created_at%22%3A%20%222015-09-15T15%3A06%3A23Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/12096004%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/miguelgaio%22%7D%7D%2C%20%7B%22body%22%3A%20%22%3A%2B1%3A%20%22%2C%20%22created_at%22%3A%20%222015-09-15T15%3A27%3A06Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/12096004%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/miguelgaio%22%7D%7D%2C%20%7B%22body%22%3A%20%22%3Ashipit%3A%20%22%2C%20%22created_at%22%3A%20%222015-09-16T12%3A31%3A47Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/6862950%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/krocard%22%7D%7D%5D%2C%20%22title%22%3A%20%22General%20Comment%22%7D%2C%20%22Pull%20665e3f14668f52801e5f9bcac479306fe25248d8%20CMakeLists.txt%203%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/01org/parameter-framework/pull/232%23discussion_r39540215%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22Update%20comment%22%2C%20%22created_at%22%3A%20%222015-09-15T17%3A35%3A06Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/6862950%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/krocard%22%7D%7D%5D%2C%20%22title%22%3A%20%22File%3A%20CMakeLists.txt%3AL27-34%22%7D%2C%20%22Pull%20665e3f14668f52801e5f9bcac479306fe25248d8%20skeleton-subsystem/CMakeLists.txt%203%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/01org/parameter-framework/pull/232%23discussion_r39540235%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22Update%20comment%20%28or%20change%20by%20%5C%22has%20been%20tested%20on%20%2Athis%20version%2A%20only%5C%22%29%22%2C%20%22created_at%22%3A%20%222015-09-15T17%3A35%3A14Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/6862950%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/krocard%22%7D%7D%5D%2C%20%22title%22%3A%20%22File%3A%20skeleton-subsystem/CMakeLists.txt%3AL27-34%22%7D%7D%2C%20%22approved%22%3A%20%7B%22https%3A//github.com/krocard%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/6862950%3Fv%3D3%22%7D%7D%7D'></a>
<a href='https://www.codereviewhub.com/'></a>

<a href='https://github.com/krocard'></a>

- [ ] <a href='#crh-comment-Pull 665e3f14668f52801e5f9bcac479306fe25248d8 CMakeLists.txt 3'></a> &nbsp;<b><a href='https://github.com/01org/parameter-framework/pull/232#discussion_r39540215'>File: CMakeLists.txt:L27-34</a></b>
- <a href='https://github.com/krocard'></a> Update comment
- [ ] <a href='#crh-comment-Pull 665e3f14668f52801e5f9bcac479306fe25248d8 skeleton-subsystem/CMakeLists.txt 3'></a> &nbsp;<b><a href='https://github.com/01org/parameter-framework/pull/232#discussion_r39540235'>File: skeleton-subsystem/CMakeLists.txt:L27-34</a></b>
- <a href='https://github.com/krocard'></a> Update comment (or change by "has been tested on *this version* only")

<a href='https://www.codereviewhub.com/01org/parameter-framework/pull/232?mark_as_completed=1'></a>&nbsp;<a href='https://www.codereviewhub.com/01org/parameter-framework/pull/232?approve=1'></a>&nbsp;<a href='https://www.codereviewhub.com/01org/parameter-framework/pull/232?approve=0'></a>&nbsp;<a href='https://github.com/01org/parameter-framework/pull/232'></a>
<a href='#crh-end'></a>

### CODE REVIEW

**Comment by dawagner** [comment]
This pull request is now managed with CodeReviewHub. <a href='https://github.com/01org/parameter-framework/pull/232'>Refresh</a>.<a href='#crh-update-none'></a>

**Comment by miguelgaio** [comment]
:+1:

**Comment by miguelgaio** [comment]
:+1:

File: CMakeLists.txt [hunk 1]
```diff
@@ -27,7 +27,7 @@
 # SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
 # has been tested on 2.8 only - might work on older versions
```
Comment (krocard) [inline CMakeLists.txt]: Update comment

File: skeleton-subsystem/CMakeLists.txt [hunk 2]
```diff
@@ -27,7 +27,7 @@
 # SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
 # has been tested on 2.8 only - might work on older versions
```
Comment (krocard) [inline skeleton-subsystem/CMakeLists.txt]: Update comment

**Comment by krocard** [comment]
:shipit:

**Pull request merged**